src/client/app/components/groups/CreateGroupComponent.tsx

```diff
@@ -61,12 +87,45 @@ class CreateGroupComponent extends React.Component<CreateGroupPropsWithIntl, {}>
 						<h3 style={centerTextStyle}>
 							<FormattedMessage id='create.group' />
 						</h3>
-						<div style={divBottomStyle}>
+					</div>
+					<div style={divFlexStyle} className='row col-6'>
+						<div style={divBottomStyle} className='col-4 float-left'>
 							<p style={textStyle}>
 								<FormattedMessage id='name' />:
 							</p>
 							<Input type='text' placeholder={this.props.intl.formatMessage(messages.name)} onChange={this.handleNameChange} />
 						</div>
+						<div style={divBottomStyle} className='col-4'>
+							<p style={textStyle}>
+								<FormattedMessage id='group.gps' />:
+							</p>
+							<Input type='text' onChange={this.handleGPSChange} />
+						</div>
+					</div>
+					<div style={divFlexStyle} className='row col-6'>
+						<div style={divBottomStyle} className='col-4 float-left'>
+							<p style={textStyle}>
+								<FormattedMessage id='displayable' />:
+							</p>
+							<Input type='select' name='displayselect' onChange={this.handleDisplayChange}>
+								<option value='true'> True </option>
+								<option value='false'> False </option>
+							</Input>
+						</div>
+						<div style={divBottomStyle} className='col-4'>
+							<p style={textStyle}>
+								<FormattedMessage id='map.note' />:
```

Comment:
Should this be map.note or group.note? Same question in EditGroupsComponent.

Reply:
I was going to use the map.note because it just says Note in general, so instead map.note is changed to note and group points to this as well

Reply:
For reasons I did not see quickly, the map page shows the header of map.note in all languages.